Trunk/IPS.Tracker.Web/Scripts/app/App.jsx

```diff
@@ -297,6 +315,78 @@ class ReleaseDate extends React.Component {
   }
 }
 
+class SelectTest extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            taskNo: ''            
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+    
+    handleChange(taskNo) {      
+
+        this.setState({
+            taskNo: taskNo.value
+            }, () => {
+            this.props.onSelectTest(this.state);
+            })        
+    }
+
+    componentDidMount() {        
+        var url = "/Home/GetDefects";
+        
+        var items = [{
+            value: '',
+            label: ''
+        }];
+
+        $.ajax({
```

Comment:
jQuery with React? Didn't we agree on using something else?

Reply:
axios?